x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_classification.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiHealth, EuiSpacer, EuiSuperSelect, EuiTitle } from '@elastic/eui';
+import d3 from 'd3';
+import { isDecisionPathData, useDecisionPathData } from './use_classification_path_data';
+import { FeatureImportance, TopClasses } from '../../../../../common/types/feature_importance';
+import { DecisionPathChart } from './decision_path_chart';
+import { MissingDecisionPathCallout } from './missing_decision_path_callout';
+
+interface ClassificationDecisionPathProps {
+  predictedValue: string | boolean;
+  predictionFieldName?: string;
+  featureImportance: FeatureImportance[];
+  topClasses: TopClasses;
+}
+
+// cast to 'True' | 'False' | value to match Eui display
+const getStr = (v: string | boolean): string =>
+  typeof v === 'boolean' ? (v ? 'True' : 'False') : v;
```

Comment:
The Kibana style guide says to avoid nested ternary operators: https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#only-use-ternary-operators-for-small-simple-code

We should also give the function a more representative name like maybe `getStringBasedClassName()`.

Reply:
With returning boolean values as strings of `True/False` like now I end up with an empty chart:

<img src="https://user-images.githubusercontent.com/230104/91161759-742bd280-e6cb-11ea-875f-8b39acb8446b.png" width="50%" />

The reason is that in `top_classes` the class name value is a raw boolean, but the class name in the feature importance array is a string of `"1"/"0"`. If I change `getStr` to return `"1"/"0"` instead the chart works:

<img src="https://user-images.githubusercontent.com/230104/91162073-e3a1c200-e6cb-11ea-9346-cd0e685df323.png" width="50%" />

But as we can see in the screenshot we lose the `True/False` label in the dropdown. So this will need some more changes to pass around the original value/type and infer what to display.

Reply:
@walterra Hi Walter, Ben merged his PR that fixes the inconsistency between `top_classes` and the `feature_importance` array here https://github.com/elastic/elasticsearch/pull/61597

I tested with the latest snapshot and everything is working as expected.

Also updated the `getStringBasedClassName` function here https://github.com/elastic/kibana/pull/73561/commits/a954cc633e4d414d7ea61ec6a6d9e36dc97d4737